### ISSUE TITLE
Release v1.4 and translation acknowledgement in Docs

### ISSUE THREAD

**Issue by pat-schmitt** [OGGM/world-glacier-explorer#37]
Should we make a new release v1.4? There are three new languages included since v1.3 (Portuguese, Luxembourgish and Hindi).

And I was thinking if we should make a sort of section `Translations made by` on https://edu.oggm.org/en/latest/explorer.html, to acknowledge the different contributions more visible.

**Comment by anoukvlug** [comment]
I like both ideas 😃

**Comment by fmaussion** [comment]
Yes! Maybe we should ask the workshop participants to ask with a couple languages not available yet: Hebrew, Croatian, something else?

**Comment by anoukvlug** [comment]
Romanian, maybe Norwegian?

**Comment by pat-schmitt** [comment]
I am currently about to add a howto_release file (similar to the glacier simulator) and I was wondering if I also should add a stable branch.
The reason for this is that the MyBinder link on the website is currently pointing to the current master -> the app version you get on the website currently is different when you start the app on the bremen server (is defined here https://github.com/OGGM/Bokeh-Docker/blob/master/bokeh.oggm.org/docker-compose.yml) or when you start it by clicking on the MyBinder link (is always opening the latest master).

**Comment by fmaussion** [comment]
sounds good!
